Will not build on android with Java 1.6

I added this plugin to my project, and it works great for iOS, but Android will no longer build. I am getting:
```
/MyProject/platforms/android/src/com/terikon/cordova/photolibrary/PhotoLibraryService.java:78: error: diamond operator is not supported in -source 1.6
    ArrayList<JSONObject> results = new ArrayList<>();
```
Am I missing something?

Thanks!

--- a/src/android/PhotoLibrary.java
+++ b/src/android/PhotoLibrary.java
@@ -318,7 +318,7 @@ public class PhotoLibrary extends CordovaPlugin {
 
   private void requestAuthorization(boolean read, boolean write) {
 
-    List<String> permissions = new ArrayList<>();
+    List<String> permissions = new ArrayList<String>();
 
     if (read) {
       permissions.add(READ_EXTERNAL_STORAGE);

--- a/src/android/PhotoLibraryService.java
+++ b/src/android/PhotoLibraryService.java
@@ -75,7 +75,7 @@ public class PhotoLibraryService {
 
     final ArrayList<JSONObject> queryResults = queryContentProvider(context, MediaStore.Images.Media.EXTERNAL_CONTENT_URI, columns, "");
 
-    ArrayList<JSONObject> results = new ArrayList<>();
+    ArrayList<JSONObject> results = new ArrayList<JSONObject>();
 
     for (JSONObject queryResult : queryResults) {
       if (queryResult.getInt("height") <=0 || queryResult.getInt("width") <= 0) {

